src/VisualStudio/Core/Test/Diagnostics/VisualStudioDiagnosticAnalyzerProviderTests.vb

```diff
@@ -60,5 +60,21 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Diagnostics
                 Assert.Equal("TestAnalyzer", analyzers(0).ToString)
             End Using
         End Sub
+
+        <Fact>
+        Public Sub GetRazorReferencesInExtensions()
+            Dim extensionManager = New VisualStudioDiagnosticAnalyzerProvider(
+                New MockExtensionManager({({"razorPath1", "razorPath2"}, "RazorVsix")}, contentType:="Microsoft.VisualStudio.RazorAssembly"),
```

Comment:
Consider also testing when there are Analyzer items (they shouldn't be returned by GetRazorReferencesInExtensions)